content/departments/marketing/blog.md

```diff
@@ -74,7 +79,29 @@ The `description` field is used as an excerpt for your post on the blog the inde
 - The `socialImage` field is optional. Use the full path to image in order to be read properly on Twitter and Facebook. Ideal image size: 1,200 x 628 pixels. <a href="https://sproutsocial.com/insights/social-media-image-sizes-guide/" rel="nofollow" target="_blank">Latest social size guidelines</a>.
 - The `videoID` field is an optional YouTube video ID and will take priority even if the `socialImage` is present. This will generate an inline video preview card when sharing on social media. This is supported for all types of posts; blog, podcast, and release posts.
 
-## Adding images and other media
+## Using the Blog Post Starter Pack
+
+The Content Platform Team maintains a template Markdown file that exposes all of the content components available for use in the blog. You can directly copy and paste components from this file into your post. It's important to use the components provided for tables, images, alerts, youtube videos, videos, and hubspot forms to ensure our blog posts are uniform in styling.
+
+Access the template in the About repository: [`/content/blogposts/blog-post-starter-pack.md`](https://github.com/sourcegraph/about/tree/main/content/blogposts/blog-post-starter-pack.md).
+
+To use the template, copy the above file and paste it in the appropriate year folder (i.e. `content/blogposts/2022/`). Rename the file to the slug the post will be, or something similar. Edit the markdown content accordingly, all available standardized components are ready to be changed right in this file!
+
+## Adding images
+
+Please use a `Figure` component to add images to your post. Optionally, you can include a `caption` to display a description below the image.
+
+<!-- prettier-ignore -->
+```html
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
+  alt="Sourcegraph thumbnail"
+  caption="The Sourcegraph logo"
+  link="https://https://sourcegraph.com/search.com"
+/>
+```
```

Comment:
Instead of referencing all the components code here in the handbook, I think we should just reference the starter pack alone and maintain that as the source of truth. Then we can avoid maintaining two places at once and avoid outdated API mismatches. If we ever change the API on the about site, the blog starter pack will enforce us to update it since it will error/warn on build time. I would then make a note here in this guide to "please reference the blog starter pack." What do you think?

Reply:
I see what you mean. So not show the syntax for HubSpot forms either? I can list all the components and point teammates to go to the Blog Post Starter Pack to see the syntax use.

Reply:
Let me know thoughts on this phrasing! [92083e9](https://github.com/sourcegraph/handbook/pull/4186/commits/92083e927792184ac38b9b139a68ecb2aa741cd4)

Reply:
Yeah I wouldn't show the code examples on the Handbook. I would also even move the frontmatter section to the blog starter pack and point people there too.